src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -513,4 +513,16 @@
   <data name="PX1007DacProperty" xml:space="preserve">
     <value>DAC property</value>
   </data>
+  <data name="PX1019Title" xml:space="preserve">
```

Comment:
@k-s-popova  please review diagnostic messages

Reply:
@SENya1990 , consider changing as follows:
PX1019Title: A DAC property field with the AutoNumber attribute must have the string type
PX1019Fix: Change the type to string
PX1020Title: For a DAC property field with the AutoNumber attribute, the string length must be greater than {0}
PX1020Fix: Make the string length sufficient for the auto-numbering

Reply:
Done, thanks!